### DISCUSSION TITLE
Removing xDrip collector from Wahoo Bolt

### DISCUSSION THREAD

**Discussion by dkshores63** [NightscoutFoundation/xDrip#2256] [Q&A] [answered]
Hey Happy Tuesday! I have manged to get my Wahoo Bolt bike Computer working with my Dexcom G6 and Displaying my BG readings! It is great! It is a long story on how I did it, but I am trying to remove it from my old Wahoo so whomever is using it next, won't have the Scanning message forever taking up space on the screen. I have used Start USB Configuration command with xdrip collector disabled as suggested in the wiki, but it doesn't work. The BG value continues to be displayed. Part of the long story mentioned above.. and suggestions?

**Comment by jamorham** [comment]
Use the on screen menus on the Wahoo to perform a factory reset - this should remove the feature.

**Reply by dkshores63** [reply]
Thanks for the suggestion.   I've tried that many times,  but just to be thorough it did another reset just now w/o success.  Still pops up as Scanning, etc.  After the reset. 8-(

**Reply by jamorham** [reply]
Try plugging it in to PC so that it appears as an external drive and delete any files there and then reboot it. See if that helps

**Comment by dkshores63** [comment, marked as answer]
Ok, great idea.  I was able to find an empty Xdrip folder and an xdrip.cfg text file in the usb storage folder.  Once I removed the xdrip.cfg file, that removed the cgm field on the display!

Thanks
       DKS

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
